packages/dx-react-scheduler-material-ui/src/templates/drag-drop/appointments.jsx

```diff
@@ -22,29 +23,35 @@ const sourceStyles = {
 
 const DraftAppointmentBase = ({
   classes, className, style,
-  data, type, ...restProps
+  data, type, leftSlice, rightSlice, ...restProps
 }) => (
   <Appointment
     className={classNames(classes.appointment, className)}
     style={style}
```

Comment:
Since the `style` prop isn't required, can we remove it and use `restProps`?

Reply:
Removed